build/docker/proxy/assets/etc/nginx/nginx.conf.template

```diff
@@ -16,6 +16,16 @@ http {
                 ALLOWED_HOSTS_TO_REPLACE
     }
 
+    map $time_iso8601 $time_iso8601_p1 {
```

Comment:
What actually is this section doing?

Reply:
These maps convert the timestamp into an ISO format with milliseconds. I couldn't find any other way of doing it in NGINX.  The default timestamp isn't in ISO format, and the default ISO format doesn't include milli seconds. A lot of people are complaining about this in the community.

Reply:
thanks